### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by kr-sarvesh**
https://redstore-onlin3.netlify.app/screens/product/product.html

added filter dropdown

### CODE REVIEW

File: screens/authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+form a {
+  font-size: 15px;
+  font-weight: bold;
+  color: var(--primary-color);
+}
+
+#Indicator {
+  width: 100px;
```
Comment (harshitpaliwal95) [inline screens/authentication/auth.css:70]: use rems

File: screens/authentication/auth.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      RegForm.style.transform = 'translateX(0px)';
+      LoginForm.style.transform = 'translateX(0px)';
+      Indicator.style.transform = 'translateX(100px)';
+    }
+    function login() {
+      RegForm.style.transform = 'translateX(300px)';
+      LoginForm.style.transform = 'translateX(300px)';
+      Indicator.style.transform = 'translateX(300px)';
+    }
+  </script>
```
Comment (harshitpaliwal95) [inline screens/authentication/auth.html:92]: should avoid internal Js

**Review by harshitpaliwal95 — COMMENTED** [review]
Ui looks clean

File: screens/authentication/auth.css (hunk 1 above)
Comment (kr-sarvesh) [reply screens/authentication/auth.css:70]: okay

**Pull request merged**